test/v3/removeLiquidity.integration.test.ts

```diff
@@ -389,6 +398,45 @@ describe('remove liquidity test', () => {
             });
         });
     });
+    describe('recovery', () => {
+        let input: RemoveLiquidityRecoveryInput;
+        beforeEach(async () => {
+            // set bptIn
+            const bptIn: InputAmount = {
+                rawAmount: parseEther('0.1'),
+                decimals: 18,
+                address: poolState.address,
+            };
+            // set the input
+            input = {
+                bptIn,
+                chainId,
+                rpcUrl,
+                kind: RemoveLiquidityKind.Recovery,
+            };
+            // the pool must be put into recovery mode for the queries & transaction to work
+            await putPoolIntoRecoveryMode(
+                client,
+                txInput.poolState,
+                txInput.testAddress,
+            );
+        });
+        test('with tokens', async () => {
+            // all preparations are done, now remove liquidity
+            const removeLiquidityOutput = await doRemoveLiquidity({
+                ...txInput,
+                removeLiquidityInput: input,
+            });
+            // assert now
```

Comment:
these comments may be a bit superfluous but just my opinion

Reply:
Agree, I have removed them.